README.md

```diff
@@ -44,37 +44,37 @@ The only point to this package is that using `PyCall` to access
 a symbolic value `x`, take its sine, then evaluate at `pi`, say:
 
 ```
-using PyCall			
-@pyimport sympy
+using PyCall
+sympy = pyimport("sympy")
 x = sympy.Symbol("x")
 y = sympy.sin(x)
-y[:subs](x, sympy.pi) |> float
+z = y.subs(x, sympy.pi)
+convert(Float64, z)
```

Comment:
`|> float` or `|> Float64` no longer works?

Reply:
Yes, surprisingly, I saw this when trying to run the README examples:

```
julia> using PyCall


julia> sympy = pyimport("sympy")
PyObject <module 'sympy' from '/Users/verzani/.julia/conda/3/lib/python3.7/site-packages/sympy/__init__.py'>

julia> x = sympy.Symbol("x")
PyObject x

julia> y = sympy.sin(x)
PyObject sin(x)

julia> z = y.subs(x, sympy.pi)
PyObject 0

julia> float(z)
ERROR: MethodError: no method matching AbstractFloat(::PyObject)
Closest candidates are:
  AbstractFloat(::Bool) at float.jl:252
  AbstractFloat(::Int8) at float.jl:253
  AbstractFloat(::Int16) at float.jl:254
  ...
Stacktrace:
 [1] float(::PyObject) at ./float.jl:271
 [2] top-level scope at none:0

```

Reply:
Oh yeah, that's because constructors no longer fall back to convert.

I tried adding the constructor methods in manually, but then I got a massive slowdown: https://github.com/JuliaLang/julia/issues/27599

I should probably add a `float` method, though.

Reply:
I don’t rely on it here, but it might be helpful. I’d like to see how much of SymPy can be trimmed out now that the getproperty feature is used. I think quite a bit.